### DISCUSSION TITLE
Support for Veikk VK1060PRO?

### DISCUSSION THREAD

**Discussion by abcasker** [OpenTabletDriver/OpenTabletDriver#2011] [General]
hello, I tried downloading the opendriver but it won't detect since it's not supported. will there be a support for Veikk VK1060PRO?

**Comment by xanndevs** [comment]
Why 1060 is supported but not 1060pro still?

**Comment by jamesbt365** [comment]
Support cannot be provided without an owner of the tablet stepping forward and assisting the developers with actually supporting the tablet. This is much better off done within our Discord, which you can find at https://opentabletdriver.net/discord
